webapp/app/components/userMenu/userMenu.component.js

```diff
@@ -0,0 +1,64 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import IconButton from 'material-ui/IconButton';
+import Menu, { MenuItem } from 'material-ui/Menu';
+import { FormattedMessage } from 'react-intl';
+
+import { UserAvatar } from '../';
```

Comment:
??

Reply:
nie importuj w ten sposob tych komponentow pomiedzy soba, moga latwo powstac circular dependencies. zaimportuj lepiej z .component

Reply:
w takim wypadku importujesz UserMenu z indexu w komponencie, ktory wewnatrz ma UserAvatar, ktory odwoluje sie do tego samego indexu

Reply:
masz tak w wielu miejscach w sumie wiec zwroc na to uwage